lib/ui/components/captured-requests.js

```diff
@@ -46,19 +46,17 @@ export class CapturedRequests extends React.Component {
   _actionButtonFor(request) {
     return this._isAlreadyMocked.bind(this, request)()
       ? <button className="btn btn-edit" onClick={ this._editRequest.bind(this, request.id) }>Edit</button>
-      : <button className="btn btn-mock" onClick={ this._mockRequest.bind(this, request) }>Mock</button>;
+      : '';
```

Comment:
Because now there's no one-to-one relationship between mocked requests and captured requests.
You can potentially have:
/api/posts/*
But still want to give a specific response to
/api/posts/15